src/common/packager.ts

```diff
@@ -162,6 +162,18 @@ export class Packager {
 
                 let spawnOptions = { env: reactEnv };
 
+                // Since expo@37, you must specify the sourceExts parameter for the normal loading of additional files.
```

Comment:
```suggestion
                // Since expo@37, you must specify the sourceExts parameter so that the packager can load additional files, such as custom fonts:
```